### PR TITLE
#5745 [Manager] The latest log should be displayed first on devTool

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5745 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/LogViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import { forEachRight, map, times } from 'lodash';
+import Skeleton from '@material-ui/lab/Skeleton';
+import Box from '@material-ui/core/Box';
+
+interface LogViewerProps {
+  logs: string[];
+  loading?: boolean;
+}
+
+function reverse(logs: string[]): string[] {
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/LogViewer.tsx:11]: Don't know if they work the same way but `Array.prototype.reverse()` is capable to reverse an array of strings :)

Comment (eechih) [reply ohara-manager/client/src/components/common/LogViewer.tsx:11]: Good suggestion, I use Array.prototype.reverse() instead :)

**Review by konekoya — APPROVED** [review]

**Comment by eechih** [comment]
retry manager-it

**Comment by eechih** [comment]
retry manager

**Pull request merged**